tests/includes/CoreLoader.php

```diff
@@ -43,23 +66,11 @@ protected function setConstants()
             if (file_exists($_tests_dir . '/includes/functions.php')) {
                 define('WP_TESTS_DIR', $_tests_dir);
             } else {
-                define(
-                    'WP_TESTS_DIR',
-                    dirname(
-                        dirname(
-                            dirname(
-                                dirname(
-                                    dirname(
-                                        dirname(
-                                            __DIR__
-                                        )
-                                    )
-                                )
-                            )
-                        )
-                    ) . '/tests/phpunit'
-                );
```

Comment:
LOL what was that? 😄

Reply:
:christmas_tree: :santa: 
jingle bells style

Reply:
that was a previous dev that at this point doesn't need to be mentioned ;)